AOEMods.Essence.Editor/ExportFileDialogViewModel.cs

```diff
@@ -16,22 +17,41 @@ public class ExportFileDialogViewModel : ObservableRecipient
     public string OutputFilePath
     {
         get => outputFilePath;
-        set => SetProperty(ref outputFilePath, value);
+        set {
+            SetProperty(ref outputFilePath, value);
+            OnPropertyChanged(nameof(IsExportEnabled));

```

Comment:
SetProperty already calls OnPropertyChanged, { should be on newline (but that won't matter once it's only using => SetProperty)

Reply:
Whoops nvm this one didn't see it uses a different name